File: src/main/java/me/skynda/classification/entities/Classification.java
package me.skynda.classification.entities;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;
import java.util.Date;

@Entity
@Data
@Table(name = "classification")
public class Classification implements Serializable{
    @Id
    @Column(name = "id", columnDefinition = "serial")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "description")
    private String description;

    @Column(name = "is_imported")
    private Boolean isImported;

    @Column(name = "weight")
    private Integer weight;

    @Column(name = "value")
    private String value;

    @Column(name = "modifier_user_id")
    private Integer modifierUserId;

    @Column(name = "modifier_user_ip")
    private String modifierUserIp;

    @Column(name = "archived")
    private Date archived;

    @Column(name = "name")
    private String name;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "classification_type_id")
    private ClassificationType classificationType;

    @Column(name = "is_active")
    private Boolean isActive;

    @Column(name = "value2")
    private String value2;

}

File: skynda_dotnet_backend/Triven.Data.EntityFramework/Models/Classification.cs
using System.ComponentModel.DataAnnotations.Schema;
using Triven.Data.EntityFramework.Models.Base;
using Triven.Domain.Models;

namespace Triven.Data.EntityFramework.Models
{
    [Table("Classification")]
    public class Classification : AuditableModel, IClassification
    {
        public string Description { get; set; }

        public bool IsImported { get; set; }

        public int Weight { get; set; }

        public string Value { get; set; }

        public string Name { get; set; }

        /// <summary>
        /// many-to-one relation of ClassificationType
        /// </summary>
        public ClassificationType ClassificationType { get; set; }

        public bool IsActive { get; set; }

        public string Value2 { get; set; }
    }
}